packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -908,8 +909,17 @@ export default class TradeStore extends BaseStore {
                             const is_digit_contract = isDigitContractType(category?.toUpperCase() ?? '');
                             const is_multiplier = isMultiplierContract(category);
                             const contract_type = category?.toUpperCase();
-
-                            if ((window as any).hj) (window as any).hj('event', `placed_${category}_trade`);
+                            const is_call = category === 'Call';
+                            const is_put = category === 'Put';
+                            const is_high_low = isHighLow({ shortcode_info: extractInfoFromShortcode(shortcode) });
+                            const higher_lower_contact = is_high_low && is_call ? 'higher' : 'lower';
+                            const rise_fall_contract = !is_high_low && is_call ? 'rise' : 'fall';
```

Comment:
```suggestion
                            const rise_fall_contract = is_call ? 'rise' : 'fall';
```

Reply:
These conditions already checked in the following line right? Either remove it from here or from there.